src/index.js

```diff
@@ -70,18 +71,21 @@ const instance = {
    * @returns {object} promise
    */
   start: () => Promise.resolve().then(() => {
+    // CLI command, execute and return early
+    const cliCommand = commands[args.raw._[0]]
+    if (cliCommand) return cliCommand()
```

Comment:
Here is where I've shimmed in support for commands.  If there is a command available, it will be executed immediately and the usual task execution circumvented.  This is a bit naive but works for now.